breathecode/marketing/tests/urls/tests_academy_lead.py

```diff
@@ -735,3 +735,178 @@ def test_academy_lead__with_ten_datas_with_location_with_comma_pagination_after_
         self.assertEqual(self.all_form_entry_dict(), [{
             **self.model_to_dict(model, 'form_entry')
         } for model in models])
+
+    """
+    🔽🔽🔽 With full like in querystring
+    """
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_academy_lead__with_full_name_in_querystring(self):
```

Comment:
I have doubts about if it work, we should add other test bases in it, in this case, we have a user called Konan Smith and other called Konan Mosley, the test should returns just one user

Reply:
Think a user can write a full name and the normal behavior should be the user just look one user if exists just one in the db